### ISSUE TITLE
Menu / Settings / Advanced craches

### ISSUE THREAD

**Issue by Badiboy** [UnigramDev/Unigram#1574]
Menu / Settings / Advanced.

![image](https://user-images.githubusercontent.com/5613421/81620667-13ff7200-93f5-11ea-923e-9d801054065a.png)

Windows 10 (latest). Unigram (today update from Store).
Before today update it was fine.

_PS. Unigram tried to launch in parallel with update and some "cannot start" error occurs. May it be the reason or not..._

**Comment by MarcAnt01** [comment]
Duplicate of #1280

**Comment by Badiboy** [comment]
In the https://github.com/UnigramDev/Unigram/issues/1280 it 

> Disappeared once I got 3.13.2649

But I got it now in 3.15.4756!
